### PR TITLE
Set no-cache header

### PR DESCRIPTION
**Pull request by zpao**
Supersedes #35 (this does all the server implementations, not just the node one).

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -81,5 +87,6 @@ func handleComments(w http.ResponseWriter, r *http.Request) {
 func main() {
 	http.HandleFunc("/comments.json", handleComments)
 	http.Handle("/", http.FileServer(http.Dir("./public")))
+	log.Println("Server started: http://localhost:3000")
```
Comment (spicyj) [inline server.go]: can't ListenAndServe fail, making this confusing? ah well.

File: server.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def comments_handler():
         with open('_comments.json', 'w') as file:
             file.write(json.dumps(comments, indent=4, separators=(',', ': ')))
 
-    return Response(json.dumps(comments), mimetype='application/json')
+    return Response(json.dumps(comments), mimetype='application/json', headers={'Cache-control': 'no-cache'})
```
Comment (spicyj) [inline server.py]: nit: `-Control` capitalized for consistency with the others?

**Comment by spicyj** [comment]
lgtm

File: server.go (hunk 1 above)
Comment (zpao) [inline server.go]: Yea, though it logs that it fails. Perhaps I can say "Starting server…" but it's not a huge deal.

```
2015/03/01 12:00:04 Server started: http://localhost:3000
2015/03/01 12:00:04 listen tcp :3000: bind: address already in use
exit status 1
```

File: server.py (hunk 2 above)
Comment (zpao) [inline server.py]: Good catch

**Pull request merged**